packages/tenant-process/src/services/tenantService.ts

```diff
@@ -150,6 +151,29 @@ export async function retrieveAttribute(
   return attribute;
 }
 
+async function retrieveTenantAttribute(
```

Comment:
Since you check for verified attribute, let's name this `retrieveTenantVerifiedAttribute` (or, in alternative, add the kind as parameter)
```suggestion
async function retrieveTenantVerifiedAttribute(
```